src/display/views/help/components/ContactFormPane.js

```diff
@@ -36,9 +27,15 @@ class ContactFormPane extends React.Component {
 		}));
 	}
 
+	submitForm() {
```

Comment:
This doesn't need to be a form submission - you can remove all the form stuff and just make it a direct link in a new tab. :) As I mentioned in my other comment, you can see `src\display\views\tools\components\BrowserExtensionsPane.js` for an example where we're already doing this.

Reply:
Yep thanks, got more info from the file mentioned. Thank you!